src/lib/Service/UserPreferences/DateTimeFormat/DateTimeFormatRepository.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Service\UserPreferences\DateTimeFormat;
```

Comment:
I think it would make more sense to put all datetime related classes in separate namespace since `EzSystems\EzPlatformAdminUi\Service` is not the best namespace for these.